contracts/swear.sol

```diff
@@ -0,0 +1,144 @@
+pragma solidity ^0.4.0;
+
+import "./abstracts/swearabstract.sol";
+import "./abstracts/trialrulesabstract.sol";
```

Comment:
so AFAIU trialrules are needed for compensation, but this makes the swear and swindle too interdependent again.
Either just pass all compensation related params from the swindle contract or 
completely source this parameter independently.
JUstiication for the latter is that sentence and penalties are conceptually indepenfent of the rules of a game

Reply:
ok.